src/main/java/net/dries007/tfc/common/items/RottenCompostItem.java

```diff
@@ -34,9 +34,9 @@ public InteractionResult useOn(UseOnContext context)
 
         if (state.getBlock() instanceof ICropBlock crop)
         {
-            if (!level.isClientSide)
+            if (!level.isClientSide && level.getBlockEntity(pos) instanceof CropBlockEntity be)
```

Comment:
`entity`, `blockEntity`, `cropEntity`, just not `be` because that's a verb, not a `CropBlockEntity`.